src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,10 +18,14 @@
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            PersonType personType = ParserUtil.parsePersonType(args);
+            String trimmed = args.trim();
+            String[] split = trimmed.split(" ");
+            // If personType or index not given, it is an invalid command format. ArrayIndexOutOfBoundsException
+            // will be thrown by these lines below but we catch it and throw a ParseException
+            Index index = ParserUtil.parseIndex(split[1]);
+            PersonType personType = ParserUtil.parsePersonType(split[0]);
             return new DeleteCommand(index, personType);
-        } catch (ParseException pe) {
+        } catch (ParseException | ArrayIndexOutOfBoundsException pe) {
```

Comment:
I think the `ArrayIndexOutOfBoundsException` should be in another catch block since its likely to have a different error message?